Documentation/project-docs/standard-platform.md

```diff
@@ -23,12 +23,14 @@ The .NET Platform Standard version represents binary portability across platform
 
 
 ## Terms
+- **.NET Platform Standard** - versioned sets of the reference assemblies that all .NET Platforms must support.
 - **PCL** - Portable Class Library
 - **Platform** - e.g. .NET Framework 4.5, Windows Phone 8.1
 - **Reference Assembly** - An assembly that contains API surface only. There is no IL in the method bodies. It is used for compilation only, and cannot be used to run. Also commonly referred to as "Contracts".
-- **Implementation Assembly** - An assembly that contains an implementation of a reference assembly. These are usually implemented in the platform itself and cannot be updated without updating the platform.
-- **.NET Platform Standard** - versioned sets of the reference assemblies.
-- **Multi-targeting** - to compile the same source code files to different target platforms, i.e. against different API sets.
+- **Implementation Assembly** - An assembly that contains an implementation of a reference assembly. These can be implemented as standalone assemblies but can sometimes be anchored by a platform and cannot be updated without updating the platform.
+- **Anchored Assembly** - An assembly that is not in the Platform Standard and is able to ship independently on top of many platforms but for specific platforms, the platform must be updated to support them. This set varies by platform.
```

Comment:
Another part of the doc seems to suggest that everything in corefx is a part of the platform standard, under the "Relationship to Platforms" header:

> It is a standard that platforms are implemented against. The .NET Platform Standard defines reference assemblies (contracts) that platforms implement. [These contracts are defined by CoreFX](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#list-of-net-corefx-apis-and-their-associated-net-standard-platform-version-tentative) and are either implemented in CoreFX as stand-alone managed code, or provided by a specific platform.

XML is in corefx, so that would make it part of the platform standard, right? Further, I'm not sure that this is descriptive enough:

> In many cases these are important things to the ecosystem but can be implemented on top

because, to a certain extent, our whole hierarchy of libraries is implemented "on top" of lower libraries, so it just depends where you specify the cut-off.